apps/fz_http/lib/fz_http/allow_rules.ex

```diff
@@ -32,6 +75,18 @@ defmodule FzHttp.AllowRules do
   end
 
   def list_allow_rules(%User{} = user) do
-    Repo.all(from r in AllowRule, where: r.user_id == ^user.uuid)
+    Repo.all(user_query(user.uuid))
```

Comment:
I guess you need to rebase to master, `uuid` field is gone :)

Reply:
Will rebase after merging this and the next PR with `fz-next`

Reply:
(To prevent more conflicts with the other PR I am building over this one)